js/jquery.twentytwenty.js

```diff
@@ -77,35 +103,41 @@
         container.addClass("active");
         offsetX = container.offset().left;
         offsetY = container.offset().top;
-        imgWidth = beforeImg.width(); 
-        imgHeight = beforeImg.height();          
+        imgWidth = beforeImg.width();
+        imgHeight = beforeImg.height();
       });
 
-      slider.on("moveend", function(e) {
+      moveTarget.on("moveend", function(e) {
         container.removeClass("active");
       });
 
-      slider.on("move", function(e) {
+      moveTarget.on("move", function(e) {
         if (container.hasClass("active")) {
-          sliderPct = (sliderOrientation === 'vertical') ? (e.pageY-offsetY)/imgHeight : (e.pageX-offsetX)/imgWidth;
-          if (sliderPct < 0) {
-            sliderPct = 0;
-          }
-          if (sliderPct > 1) {
-            sliderPct = 1;
-          }
+          sliderPct = getSliderPercentage(e.pageX, e.pageY);
           adjustSlider(sliderPct);
         }
       });
-      
-      slider.on("touchmove", function(e) {
+
+      moveTarget.on("touchmove", function(e) {
         e.preventDefault();
       });
 
       container.find("img").on("mousedown", function(event) {
         event.preventDefault();
       });
 
+      if (options.click_to_move) {
+        moveTarget.on('click', function(e) {
```

Comment:
should this be `container` instead of `moveTarget`?

Reply:
Indeed, I fixed and pushed the updated.